domain/usecase/src/main/java/com/cupcake/usecase/job/GetPopularJobsUseCase.kt

```diff
@@ -0,0 +1,23 @@
+package com.cupcake.usecase.job
+
+import com.cupcake.models.JobTitle
+import repo.JobFinderRepository
+import javax.inject.Inject
+
+class GetPopularJobsUseCase @Inject constructor(
+    private val repository: JobFinderRepository
+) {
+
+    suspend operator fun invoke(limit: Int): List<JobTitle> {
```

Comment:
this function no need more logic here in backend provide PopularJobs depend data in database